server/services/offendersService.js

```diff
@@ -324,6 +324,68 @@ module.exports = function createOffendersService(nomisClientBuilder, formService
     }
   }
 
+  async function getUpcomingReferredOffenders(context, transactionalDbClient) {
+    const agencyId = context.user.activeCaseLoad.caseLoadId
+    try {
+      const nomisClient = nomisClientBuilder(context)
+
+      const securityReferred = await formService.getSecurityReferrals(agencyId, transactionalDbClient)
```

Comment:
Did we want to restrict these to just ones that are 'NEW' and 'REFERRED'?

Reply:
Also did we want to display the **Status** on the final page instead of the **Days since sentence**?

Reply:
Its just new (as it is only for offenders whose categorisation hasnt started yet). So that means status is not relevant either 😄 .
And also , yes, I dont think 'Days since sentence' is worth showing either.

Reply:
On second thoughts, the security user can flag someone who requires an initial categorisation, (sorry I didnt think of this) .. so I think we will have to show the same 'Days since sentence' 'Date due' and 'type' columns as the security home page list for these,